libs/core/src/lib/card/card-second-subtitle.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit, ChangeDetectionStrategy, ElementRef, Renderer2 } from '@angular/core';
+
+import { CLASS_NAME } from './constants';
+
+@Component({
```

Comment:
Check my comment regarding title/subtitle

Reply:
> If it's only one class you can use event @HostBinding(), but I would like to hear the reason why renderer is used there.
`@HostBinding()` creates a binding that should be watched by angular, so if want to add a static class to the host element only we can do it by renderer, that's more efficinet way.

Reply:
So performace purposes, that's fine for me

Reply:
I changed it to `CssClassBuilder`